runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/runtime/Platform.java

```diff
@@ -410,7 +412,15 @@ public static boolean isMac() {
 	public static final String ARCH_AARCH64 = "aarch64";//$NON-NLS-1$
 
 	/**
-	 * Constant string (value {@code aarch64} indicating the platform is running on
+	 * Constant string (value {@code ppc64le}) indicating the platform is running on
+	 * an AARCH64bit-based architecture.
```

Comment:
I don't think this AARCH64

Reply:
Yes, that's a mistake from https://github.com/eclipse-platform/eclipse.platform/pull/1575#discussion_r1785000606.

Reply:
Just to be sure you fixed the value but added AARCH64bit-based (wrong!) where it should be PPC64LEbit-based in this commit.

Reply:
Ah, yes. I oversaw that. Thanks.
Fixed it now.